classes/lable.rb

```diff
@@ -0,0 +1,15 @@
+class Label
+  attr_accessor :title, :color, :items, :id
+
```

Comment:
id and items  are read-only so I would suggest  using `attr_reader :id, :items` as indicated in the UML diagram
![ruby](https://github.com/ambrose-kibet/Catalog-of-my-things/assets/95437850/f4d289fb-bb9b-486a-bd09-2d61565200cb)